### DISCUSSION TITLE
Grey box graphic sitting behind the default handle graphic

### DISCUSSION THREAD

**Discussion by chuank** [jaames/iro.js#232] [General]
Hello everyone, I'm seeing a weird grey box graphic sitting behind the default handle graphic in my setup. The codepen examples don't exhibit this. Has anyone come across this, along with a solution?

This is what the handle look like:
![image](https://user-images.githubusercontent.com/1781911/166409956-4b87a9c1-584b-4b13-aa4e-d4a76e6f974d.png)

Just using the defaults, so not doing any custom svg handles.

(using Vue.js, but the iro element is currently just sitting as a plain HTML div)

**Comment by jaames** [comment]
Are you sure you don't have any CSS styles on your page that might be interfering with SVG elements? There isn't any reason this would happen unless your page has some styles like `svg { background: grey; }` somewhere. Inspect element on the handle would be able to track this down.

**Comment by chuank** [comment]
Thanks @jaames that was it – indeed I had another remnant CSS style overriding the background in the svg tag.
